package.json

```diff
@@ -10,8 +10,9 @@
     "backstop:approve": "node test/approve-conflicts.js",
     "backstop:init": "node test/create-references.js",
     "bootstrap:build": "node npm-scripts/build-bootstrap-workspace.js",
-    "build-patternfly": "rm -rf dist && rm -rf tmp && gulp build",
+    "build-patternfly": "rm -rf dist && rm -rf tmp && npm run build:icons && gulp build",
     "build": "gatsby build",
+    "build:icons": "node ./src/icons/generateIcons.js",
```

Comment:
File paths are case sensitive - please change to src/icons in the actual name to be consistent with the rest of our naming.

Reply:
Probly a problem when I moved stuff around.

Reply:
+1 - i think if we can resolve this and rename to `/src/Icons/generateIcons` path the PR is acceptable now.